### PR TITLE
Promises

### PR DESCRIPTION
**Pull request by illiaChaban**
mind seeing it ? @rcackerley

### CODE REVIEW

File: coffeeRunner/coffeerun-starter/scripts/main.js [hunk 1]
```diff
@@ -8,13 +8,40 @@ var repopulateButton = document.querySelector('button.btn.btn-default.right');
...
+// });
+
+// var getOrders = function() {
+//     var promise = new Promise(function(resolve) {
+//         $.get(url, resolve);
+//     })
+//     return promise;
+// }
+
+var getOrders = new Promise( function(resolve) {
```
Comment (rcackerley) [inline coffeeRunner/coffeerun-starter/scripts/main.js]: Looks good to me. Seems very clean and makes complete sense for your implementation 👏🏻

File: coffeeRunner/coffeerun-starter/scripts/main.js [hunk 2]
```diff
@@ -8,13 +8,40 @@ var repopulateButton = document.querySelector('button.btn.btn-default.right');
...
+    $.get(url, resolve);
+})
+
+var gotPromise = getOrders
```
Comment (rcackerley) [inline coffeeRunner/coffeerun-starter/scripts/main.js]: Could you do getOrders.then instead? Is creating gotPromise a necessary line of code?

File: coffeeRunner/coffeerun-starter/scripts/main.js (hunk 1 above)
Comment (illiaChaban) [reply coffeeRunner/coffeerun-starter/scripts/main.js]: yes, I could! good catch, I didn't think of that

File: coffeeRunner/coffeerun-starter/scripts/main.js (hunk 2 above)
Comment (illiaChaban) [reply coffeeRunner/coffeerun-starter/scripts/main.js]: yes, I could! good catch, I didn't think of that

File: coffeeRunner/coffeerun-starter/scripts/main.js (hunk 1 above)
Comment (illiaChaban) [reply coffeeRunner/coffeerun-starter/scripts/main.js]: thanks!

**Pull request merged**